mkt/account/serializers.py

```diff
@@ -129,6 +129,11 @@ def get_permissions(self, obj):
             'website_submitter': allowed('Websites', 'Submit'),
             'stats': allowed('Stats', 'View'),
             'revenue_stats': allowed('RevenueStats', 'View'),
+            'content_tools': {
+                'login': allowed('ContentTools', 'Login'),
+                'addon_submit': allowed('ContentTools', 'AddonSubmit'),
```

Comment:
say we wanted to open this up to foxfooders, what would this look like? would we have to add each person to the group?